.github/workflows/workflow.yml

```diff
@@ -45,6 +45,8 @@ jobs:
 
       - run: opam exec -- make build
 
+      - run: opam exec -- dune build bench/ examples/ cohttp-curl-async/bin/ cohttp-curl-lwt/bin/
```

Comment:
What additional targets does this add over `make build`?

Reply:
removed.